### PR TITLE
Add more simple tests

### PR DESCRIPTION
**Pull request by ChrisJefferson**
Some simple tests, in particular making sure we hit all lines where we InstallMethod in a single line, as they may (shortly) not be profiled correctly.

### CODE REVIEW

**Comment by markuspf** [comment]
It';s clearly comparing against https://codecov.io/gh/gap-system/gap/branch/master/commits?from=2018-08-30%2000%3A00%3A00&to=2018-09-02%2000%3A00%3A00 not the "real" coverage of `master`.

File: tst/testinstall/kernel/object.tst [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+[ "LENGTH" ]
+gap> KnownPropertiesOfObject([3,2]);
+[ "IsFinite", "IsSmallList" ]
+gap> KnownTruePropertiesOfObject([3,2]);
+[ "IsFinite", "IsSmallList" ]
+gap> KnownAttributesOfObject(Group((1,2,3)));
+[ "GeneratorsOfMagmaWithInverses", "MultiplicativeNeutralElement" ]
+
+# Don't want to hard wire this list as it is too prone to change
+gap> p := KnownPropertiesOfObject(Group((1,2,3)));
```
Comment (fingolfin) [inline tst/testinstall/kernel/object.tst]: Well... that also means this test is prone to fail, which could become quite annoying over time.

At the very least, I'd change this to print the list after sorting, and perhaps with one string per line; that way, diffs are smaller and easier to understand. I.e.:
```
gap> p := KnownPropertiesOfObject(Group((1,2,3)));;
gap> Perform(Set(p), Display);
CanEasilyCompareElements
CanEasilySortElements
IsAssociative
IsCommutative
IsCompletelyRegularSemigroup
IsCompletelySimpleSemigroup
IsCyclic
IsDuplicateFree
IsEmpty
IsFinite
IsFinitelyGeneratedGroup
IsFreeAbelian
IsGeneratorsOfMagmaWithInverses
IsGeneratorsOfSemigroup
IsGroupAsSemigroup
IsInfiniteAbelianizationGroup
IsInverseSemigroup
IsMonoidAsSemigroup
IsMonomialGroup
IsNilpotentByFinite
IsNilpotentGroup
IsNonTrivial
IsOrthodoxSemigroup
IsPerfectGroup
IsPolycyclicGroup
IsRegularSemigroup
IsSimpleSemigroup
IsSolvableGroup
IsSubsetLocallyFiniteGroup
IsSupersolvableGroup
IsTorsionFree
IsTrivial
KnowsHowToDecompose
```

But I also worry that this test will be affected heavily by whether packages are loaded or now... Do we *really* need to print this list?

Comment (ChrisJefferson) [reply tst/testinstall/kernel/object.tst]: Sorry, I meant to double semi-colon both of those, but forgot and didn't check my patch carefully before submitting. The plan is the next two tests check some properties (and true properties) which I trust will hang around and not change.

**Comment by ChrisJefferson** [comment]
I have tweaked the tests to (hopefully) be much more resiliant to changes. Sorry, I did a last minute rewriteToFile, and didn't carefully check the output, which was very bad.

**Pull request merged**